journal.py

```diff
@@ -64,12 +65,27 @@ def write(cls, title=None, text=None, session=None):
         session.add(instance)
         return instance
 
+    @classmethod
+    def update(cls, id_=None, title=None, text=None):
+        instance = cls.by_id(id_)
+        instance.title = title
+        instance.text = text
+        return instance
+
 
 def init_db():
     engine = sa.create_engine(DATABASE_URL, echo=False)
     Base.metadata.create_all(engine)
 
 
+def apply_markdown(text):
+    return markdown.markdown(
+        text,
+        extensions=['codehilite', 'fenced_code'],
+        ouput_format="html5"
+    )
+
+
```

Comment:
If this is a method on the entry class that gets `self` as its first argument, then you can simply call `entry.apply_markdown()`, right?  That'd be a bunch cleaner.